.lintstagedrc

```diff
@@ -3,9 +3,14 @@
     "prettier --write",
     "git add",
   ],
+  "*.css": [
+    "yarn run lint:css:fix",
+  ],
   "*.js": [
     "yarn run lint:css:fix",
-    "yarn run lint:js:fix",
-    "yarn run test -- --findRelatedTests",
-  ]
+    "yarn run lint:js:fix"
+  ],
+  "*.test.js": [
```

Comment:
If we are doing this, then the test cases should run when JS files are changed

Reply:
Ah, you are right.